jmix-search/search/src/test/java/test_support/TestAbstractBulkRequestActionValidationData.java

```diff
@@ -57,7 +54,7 @@ public int hashCode() {
 
     @Override
     public String toString() {
-        return String.format("Operation=%s Index=%s ID=%s", getOperationType(), index, id);
+        return String.format("Operation=Bulk Index=%s ID=%s", /*getOperationType(), */index, id);
```

Comment:
There is no operation 'Bulk'.
Bulk request is a set of single actions which can be different types.
Revert this.

If usage of DocWriteRequest.OpType prevents it from being in the base module you can create some new operation types enum and use it here.